include/stellar/verification/banded_global_extend.hpp

```diff
@@ -29,7 +29,10 @@ verifySwiftHit(Segment<Segment<TSequence const, InfixSegment>, InfixSegment> con
     // define a scoring scheme
     typedef int TScore;
     TScore match = 1;
-    TScore mismatchIndel = (TScore)_max((TScore) ceil(-1/eps) + 1, -(TScore)length(host(infH)));
+    TScore mismatchIndel = -(TScore)1000;
+    if (eps > 0.001)   // avoid division by 0
+        mismatchIndel = (TScore)_max((TScore) ceil(-1/eps) + 1, -(TScore)length(host(infH)));
```

Comment:
As the error rate decreases the scoreDropOff increases (negative correlation). Very large scoreDropOff values (i.e very small error rates) lead to extending the seed further.